notebooks/100_datamodules/103_folder.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
Are the changes in this notebook intentional? If so, not sure if `C:\\Users\\REH\\anomalib\\datasets\\MVTec\\hazelnut\\test\\crack\\` makes sense?